src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -404,6 +404,7 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
 
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
NullConditionalMemberAccess.?. operator tests.Use ?. on a dynamic method name that does not exist
```powershell
Expected an exception, with FullyQualifiedErrorId 'Argument' to be thrown, but the FullyQualifiedErrorId was 'MethodNotFound'. from /Users/runner/runners/2.160.0/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:343 char:15
    +             { (Get-Date '11/11/2019')?.$methodName() } | Should -Thro ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1: line 343
343:             { (Get-Date '11/11/2019')?.$methodName() } | Should -Throw -ErrorId 'Argument'
```